src/main/res/values/strings.xml

```diff
@@ -702,6 +702,8 @@
     <string name="pref_screen_security_explain">Request to block screenshots in the recents list and inside the app</string>
     <string name="pref_screen_security_please_restart_hint">To apply the screen security setting please restart the app.</string>
     <string name="pref_notifications">Notifications</string>
+    <string name="pref_mention_notifications">Notify mentions</string>
+    <string name="pref_mention_notifications_explain">Notify messages in muted chats if they are directed to you, like replies or reactions to your messages</string>
```

Comment:
this is 3+ lines explanations, maybe we can get shorter;

```suggestion
    <string name="pref_mention_notifications">Mentions</string>
    <string name="pref_mention_notifications_explain">In muted groups, notify messages directed to you, like replies or reactions</string>
```

"notify" in the title seems redundant.

not sure about the group/chat